src/main/app/README.md

```diff
@@ -2,25 +2,29 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.7.
```

Comment:
Is this still relevant?

Reply:
Yes and No; could be of interest to have an idea about the Webpack bundler etc stuff if at any time there is some things happen. Of course there is the abundance of packages. I personally do like to know where the origine is coming from.